qiskit incompatible with networkx 2.4

Networkx version 2.4 was released 1h ago and led to test failures on #2366.

Steps to reproduce:

```
$ pip install networkx==2.4
$ pytest cirq/circuits/qasm_output_test.py
[...watch tests fail...]
$ pip install networkx==2.3
$ pytest cirq/circuits/qasm_output_test.py
[...watch tests succeed...]
```

Test failure looks like this:

```
cirq/circuits/qasm_output_test.py ...............F..                                                                                                      [100%]

=========================================================================== FAILURES ============================================================================
______________________________________________________________ test_output_unitary_same_as_qiskit _______________________________________________________________

    def test_output_unitary_same_as_qiskit():
        qubits = tuple(_make_qubits(5))
        operations = _all_operations(*qubits, include_measurements=False)
        output = cirq.QasmOutput(operations, qubits,
                                 header='Generated from Cirq',
                                 precision=10)
        text = str(output)
    
        circuit = cirq.Circuit(operations)
        cirq_unitary = circuit.unitary(qubit_order=qubits)
>       cq.assert_qiskit_parsed_qasm_consistent_with_unitary(text, cirq_unitary)

cirq/circuits/qasm_output_test.py:289: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
cirq/testing/consistent_qasm.py:120: in assert_qiskit_parsed_qasm_consistent_with_unitary
    backend=qiskit.Aer.get_backend('unitary_simulator'))
../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/wrapper/_wrapper.py:321: in execute
    skip_transpiler)
../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/wrapper/_wrapper.py:278: in compile
    pass_manager)
../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/transpiler/_transpiler.py:74: in compile
    dags = _circuits_2_dags(circuits)
../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/transpiler/_transpiler.py:114: in _circuits_2_dags
    dags = parallel_map(DAGCircuit.fromQuantumCircuit, circuits)
../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/transpiler/_parallel.py:87: in parallel_map
    return [task(values[0], *task_args, **task_kwargs)]
../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/dagcircuit/_dagcircuit.py:1349: in fromQuantumCircuit
    dagcircuit.add_qreg(register.name, len(register))
../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/dagcircuit/_dagcircuit.py:169: in add_qreg
    self._add_wire((name, j))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <qiskit.dagcircuit._dagcircuit.DAGCircuit object at 0x7f1ddb1d3048>, name = ('q', 0), isClassical = False

    def _add_wire(self, name, isClassical=False):
        """Add a qubit or bit to the circuit.
    
            name is a (string,int) tuple containing register name and index
            This adds a pair of in and out nodes connected by an edge.
            """
        if name not in self.wire_type:
            self.wire_type[name] = isClassical
            self.node_counter += 1
            self.input_map[name] = self.node_counter
            self.node_counter += 1
            self.output_map[name] = self.node_counter
            in_node = self.input_map[name]
            out_node = self.output_map[name]
            self.multi_graph.add_edge(in_node, out_node)
>           self.multi_graph.node[in_node]["type"] = "in"
E           AttributeError: 'MultiDiGraph' object has no attribute 'node'

../../.virtualenvs/cirq-new/lib/python3.7/site-packages/qiskit/dagcircuit/_dagcircuit.py:194: AttributeError
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -3,7 +3,7 @@
 dataclasses
 google-api-python-client~=1.6
 matplotlib~=3.0
-networkx~=2.1
+networkx==2.3
 numpy~=1.16
 pandas
 protobuf==3.8.0

